src/lib/useImperialUnits.js

```diff
@@ -0,0 +1,5 @@
+import { currentLocales } from './i18n';
+
+export default function useImperialUnits() {
+  return currentLocales[0] === 'en' || Boolean(currentLocales[0].match(/(UK|US)$/));
```

Comment:
This could also be used on the leaflet map.
-1: Canada seems to be mostly using inch for length measurements - see https://en.wikipedia.org/wiki/Metrication_in_Canada#Engineering and following

Reply:
'Canadians are exposed to both metric and imperial units, and it is not unusual for there to be references to both metres and feet, acres and hectares, or grams and ounces, in the same conversation.' suggests we need an additional flag for countries that use both 😅Added CA as imperial now.